hknweb/candidate/models.py

```diff
@@ -35,7 +36,25 @@ class Meta:
     confirmed       = models.BooleanField(default=False)
     request_date    = models.DateTimeField(auto_now_add=True)
 
-    # TODO not sure if I need a field for partners
-
     def __str__(self):
         return self.name
+
+
+# CS 61A LECTURE NUMBER 3141592653589793238462643383
+class Announcement(models.Model):
+    """
+    Model for an announcement. Created by VP or some other superuser.
+    Displayed on the candidate portal. The title will be displayed in bold,
+    and the text will follow that in normal font, with a space in between.
+    """
+
+    id              = models.AutoField(primary_key=True)
```

Comment:
If possible, removing this allows Django to auto-generate the id field.
(You might need to makemigrations again to make this work.)